js_modules/dagster-ui/packages/ui-core/src/insights/InsightsBarChart.tsx

```diff
@@ -154,6 +156,10 @@ export const InsightsBarChart = (props: Props) => {
 
   const options: ChartOptions<'bar'> = React.useMemo(() => {
     return {
+      // dish: Disable animation until I can figure out why it's acting crazy.
+      // When rendering a responsive grid of charts, it's doing a distracting
+      // zoom-like animation on initial render.
+      animation: false,
```

Comment:
maybe set it to true after the initial render?

Reply:
though I realize I did the same thing 😅  https://github.com/dagster-io/internal/blob/7ce5e0c360ee21af5874bf99e97a1f028985099f/dagster-cloud/js_modules/app-cloud/src/settings/usage/UsageRoot.tsx#L64

Reply:
I did battle with this thing for a while 😆 and could not figure out what it wanted from me